src/language/variables.md

```diff
@@ -38,6 +38,48 @@ String name = 'Bob';
   of using `var`, rather than type annotations, for local variables.
 {{site.alert.end}}
 
+## Null safety
+
+The Dart language enforces sound null safety.
+
+Null safety prevents an error that results from unintentional access
+of variables set to `null`. The error is called a null dereference error.
+If you try to access a property or call a method on a variable set to `null`,
+that causes a null dereference error.
+For example, say you want to find the absolute value of an `int` variable `i`.
+If `i` is `null`, calling `i.abs()` causes a null dereference error.
+In other languages, trying this could lead to a runtime error,
+but Dart's compiler prohibits these actions.
+Therefore, Dart apps can't cause runtime errors.
+
+Null safety introduces two key changes to type declaration.
```

Comment:
"declaration" -> "declarations". (Unless you mean "declaration" as a verb, but that's a pretty uncommon use in this context.)

We tend to use "type annotations" to refer to what you're talking about here. "Type declaration" sounds like you're declaring a new type, like:

```dary
typedef Inches = int;
```

This is what I'd call a "type declaration".